Command/ExtractTranslationCommand.php

```diff
@@ -35,26 +37,20 @@
  */
 class ExtractTranslationCommand extends Command
 {
-    /**
-     * @var ConfigFactory
-     */
+    /** @var ConfigFactory */
     private $configFactory;
 
-    /**
-     * @var Updater
-     */
+    /** @var Updater */
     private $updater;
 
-    /**
-     * @var array
-     */
+    /** @var array */
     private $locales;
 
     public function __construct(ConfigFactory $configFactory, Updater $updater, array $locales)
     {
         $this->configFactory = $configFactory;
-        $this->updater = $updater;
-        $this->locales = $locales;
+        $this->updater       = $updater;
```

Comment:
i remember having disabled this in jms serializer...

Reply:
You're right, I missed some excluded rules